gocd/templates/jsonnetfile.json

```diff
@@ -8,7 +8,7 @@
           "subdir": "libs"
         }
       },
-      "version": "v1.1.9"
+      "version": "v1.2.2"
```

Comment:
Does this version version bump cause `relay-customer-1.yaml` to be generated?

Reply:
yes it does